# The ArUco marker ID to look for
uint32 target_id
uint32 num_samples

---
# Response
aruco_marker_position marker_position